Gems/PhysX/Code/Source/PhysXCharacters/Components/CharacterControllerComponent.h

```diff
@@ -63,9 +69,8 @@ namespace PhysX
             required.push_back(AZ_CRC_CE("TransformService"));
         }
 
-        static void GetDependentServices(AZ::ComponentDescriptor::DependencyArrayType& dependent)
+        static void GetDependentServices([[maybe_unused]] AZ::ComponentDescriptor::DependencyArrayType& dependent)
```

Comment:
I think if this function is empty now you can just remove it

Reply:
I found it actually easier to go through this code, review it, compare it and modify it when all the functions were present. Unless there are strong feelings against it, for this case I'd keep them.